src/pages/documentation/mobile-core/release-notes.md

```diff
@@ -164,6 +164,10 @@ The brand new Adobe Experience Platform Core Android SDKs are live! It is [open
 * Signal 2.0.0
 * Lifecycle 2.0.0
 
+<InlineAlert variant="info" slots="text"/>
+
+The `registerExtension` API for each extension is deprecated in the latest version of the mobile SDK. You don't need to register extensions separately, now you can call `MobileCore.registerExtensions` API instead. See the [Migration Guide](https://developer.adobe.com/client-sdks/previous-versions/documentation/migrate-to-android) for more details.
```

Comment:
"You can call the `MobileCore.registerExtensions` API instead of registering the extensions separately."

Also is there a reason you're not using a relative link instead of linking to the website?

Reply:
One more - "... of the Mobile SDK."

Reply:
I grabbed this text directly from our migration guide

Reply:
I am using the direct link because that was used in our release notes for Feb 23 which included all of the Android 2.0 updates. Wasn't sure if there was a reason we went with it so i did it as well.

Reply:
Got it!

I just realized I'm silly and that that's a previous versions link, so that's a-okay to link to like that.

However, the other change I suggested is still outstanding.

Reply:
Got it, fixed.